### PR TITLE
INT-4361 - align pull secret name in nxrm helm3 chart

### PR DESCRIPTION
**Pull request by kellyrob99**
- simple alignment based on spelling
- remove superfluous image-pull-secret.yaml file

### CODE REVIEW

**Review by collinpeters — APPROVED** [review]
+1

File: charts/nexus-repository-manager/templates/deployment.yaml [hunk 1]
```diff
@@ -48,7 +48,7 @@ spec:
       hostAliases:
 {{ toYaml .Values.nexus.hostAliases | indent 8 }}
       {{- end }}
-      {{- if .Values.nexus.imagePullSecret }}
+      {{- if .Values.nexus.imagePullSecrets }}
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
```
Comment (jflinchbaugh) [inline charts/nexus-repository-manager/templates/deployment.yaml:53]: did you get to play with this much? does this cram the entire nexus.imagePullSecrets into this block? I guess that allows either base64 blocks or a `name:` field (like requested in the github issue? I think I sort of understand what a sample format might be in the value.yaml, but I could use a hint or 2 to be sure.

Comment (jflinchbaugh) [reply charts/nexus-repository-manager/templates/deployment.yaml:53]: ^^ like any other of these requested features, I've not used it either, so I don't know yet.

Comment (jflinchbaugh) [reply charts/nexus-repository-manager/templates/deployment.yaml:53]: I think from the issue, I might have expected maybe just a single `imagePullSecretName' that rendered as that list of 1  `name:` field to reference to it. 
this doc, and the suggestion in the GH ticket seem to suggest we'd want to load the secret separately, and reference it by name in the chart here.
https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/

Comment (jflinchbaugh) [reply charts/nexus-repository-manager/templates/deployment.yaml:53]: Is it more versatile to do it the way you did here?

Comment (kellyrob99) [reply charts/nexus-repository-manager/templates/deployment.yaml:53]: > Is it more versatile to do it the way you did here?

Don't know if it's more versatile or not, all it does is reference a secret by name, which can then be populated in a variety of ways (base64 data or an Object with specific fields).

**Comment by kellyrob99** [comment]
https://github.com/sonatype/helm3-charts/issues/44

**Pull request merged**